google-cloud-clients/google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/MessageDispatcher.java

```diff
@@ -455,6 +455,10 @@ void extendDeadlines() {
         int sec = Math.max(1, (int) now.until(totalExpiration, ChronoUnit.SECONDS));
         modacks.add(new PendingModifyAckDeadline(sec, ackId));
         count++;
+      } else {
+        // TODO: How to get the byte count?
```

Comment:
I think the best way to do this is to make `pendingMessages` map the ack ID to `AckHandler`. Then add the `totalExpiration` field to `AckHandler`. `AckHandler` already knows how big the message is, so we can use that here.